app/src/main/java/com/swent/assos/ui/screens/Overview.kt

```diff
@@ -0,0 +1,56 @@
+package com.swent.assos.ui.screens
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.PlayArrow
+import androidx.compose.material3.Divider
+import androidx.compose.material3.ListItem
+import androidx.compose.material3.ListItemDefaults
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import com.swent.assos.model.data.Association
+import com.swent.assos.model.view.OverviewViewModel
+
+@Composable
+fun Overview(overviewViewModel: OverviewViewModel) {
+  val associations = overviewViewModel.allAssociations.collectAsState()
```

Comment:
Nitpick: When doing UI, in the PR you could try to paste some screenshot of the emulator to show what the compsoable shows on screens.